spec.html

```diff
@@ -513,8 +513,24 @@ <h1>The Syntactic Grammar</h1>
 
     <emu-clause id="sec-grammar-notation" namespace="grammar-notation">
       <h1>Grammar Notation</h1>
-      <p>Terminal symbols are shown in `fixed width` font, both in the productions of the grammars and throughout this specification whenever the text directly refers to such a terminal symbol. These are to appear in a script exactly as written. All terminal symbol code points specified in this way are to be understood as the appropriate Unicode code points from the Basic Latin range, as opposed to any similar-looking code points from other Unicode ranges. A code point in a terminal symbol cannot be expressed by a `\\` |UnicodeEscapeSequence|.</p>
-      <p>Nonterminal symbols are shown in <i>italic</i> type. The definition of a nonterminal (also called a &ldquo;production&rdquo;) is introduced by the name of the nonterminal being defined followed by one or more colons. (The number of colons indicates to which grammar the production belongs.) One or more alternative right-hand sides for the nonterminal then follow on succeeding lines. For example, the syntactic definition:</p>
+      <p>The nonterminal symbols of all the ECMAScript grammars are shown in <i>italic</i> type.</p>
+      <p>In all the grammars, 'verbatim' terminal symbols are shown in `fixed-width` font. These are to appear in a source text exactly as written. All terminal symbol code points specified in this way are to be understood as the appropriate Unicode code points from the Basic Latin range, as opposed to any similar-looking code points from other Unicode ranges. A code point in a terminal symbol cannot be expressed by a `\\` |UnicodeEscapeSequence|.</p>
```

Comment:
Somewhat of a tangent: I wish we'd pull `\` in to `UnicodeEscapeSequence` so that we could refer to it as a single production here. We do it with `LineContinuation`, and we can do it here too.

Reply:
Note that there are 18 nonterminals with 'Escape' in the name, all with the sense of something that comes *after* a backslash, i.e. the initial backslash is never included. So I'd advise against pulling the backslash into `UnicodeEscapeSequence` per se. However, we could define, e.g.
```
UnicodeBackslashSequence  :: `\` UnicodeEscapeSequence
```
(Agreed that it's a tangent.)